### DISCUSSION TITLE
file Location Format

### DISCUSSION THREAD

**Discussion by ghost** [enzo1982/freac#518] [Help & Support]
Hello Users and Support,
Everything is working as it should but I can't change the save file the way I'd like to.
Folder(artist name) > Folder(album title) > track-number_song-title_artist-name_album-title_genre.mp3
I've been converting CDs to MP3s for 25 plus years and have found a few hundred CDs I didn't have in my collects at Goodwill and the option to encode only that I can see is: `<albumartist> - <album>/<track> - <artist> - <title>`
to make edits will take hours to change manually.
How do I make edits to get what I need?
Many of the artists have multiple albums and to sort them by artist-name_album title isn't acceptable for my local storage, 
Any suggestions on how to get what I want is appreciated

**Comment by enzo1982** [comment]
Hi, you are not limited to the output filename patterns from the drop-down list. You can freely edit the pattern to create your own.

In your case, try this: `<albumartist>/<album>/<track>_<title>_<artist>_<album>_<genre>`

You can find more information about the available placeholders [here](https://freac.org/manual/en/howto.html#anchor_pat).

**Comment by ghost** [comment]
thanks Enzo1982, that works

**Reply by Lee-Carre** [reply]
@440music

Would you mark enzo1982's reply as the answer, so that this question appears as answered in the list of discussions. Thankyou.
